src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```